feature-libs/checkout/components/components/place-order/place-order.component.ts

```diff
@@ -46,10 +47,15 @@ export class PlaceOrderComponent implements OnInit, OnDestroy {
     protected fb: FormBuilder,
     protected checkoutReplenishmentFormService: CheckoutReplenishmentFormService,
     protected launchDialogService: LaunchDialogService,
-    protected vcr: ViewContainerRef
+    protected vcr: ViewContainerRef,
+    protected featureModules: FeatureModulesService
   ) {}
 
   submitForm(): void {
+    if (this.featureModules.isConfigured(ORDER_FEATURE)) {
+      this.featureModules.resolveFeature(ORDER_FEATURE).subscribe();
```

Comment:
To avoid a race condition, we should wait unitl this observable emits, and only then call `this.checkoutService.placeOrder()`

Reply:
It would be also good to add a comment explaining why we this workaround, as it's not obvious: 
```
// TODO(#14649) `OccCheckoutAdapter.placeOrder()` needs `ORDER_NORMALIZER` to be provided, but it's provided only in the lazy-loaded ORDER_FEATURE
```